.editorconfig

```diff
@@ -22,7 +23,6 @@ insert_final_newline = ignore
 trim_trailing_whitespace = false
 
 [*.{cs,cshtml,csx}]
-end_of_line = crlf
```

Comment:
I was wrong to remove this line before. Although personally I don't like CRLF line endings, since Admin API has been enforcing them in the past, we should continue enforcing them. I will restore this.